kafka-rest-common/src/main/java/io/confluent/kafkarest/Errors.java

```diff
@@ -54,11 +52,21 @@ public static RestException partitionNotFoundException() {
   public static final String CONSUMER_INSTANCE_NOT_FOUND_MESSAGE = "Consumer instance not found.";
   public static final int CONSUMER_INSTANCE_NOT_FOUND_ERROR_CODE = 40403;
 
+  /**
+   * Exception thrown when an operation tries to access a non-existing consumer instance.
+   */
+  public static final class ConsumerInstanceNotFoundException extends RestNotFoundException {
+
+    private ConsumerInstanceNotFoundException() {
+      super(CONSUMER_INSTANCE_NOT_FOUND_MESSAGE, CONSUMER_INSTANCE_NOT_FOUND_ERROR_CODE);
+    }
+  }
+
+  /**
+   * @see ConsumerInstanceNotFoundException
+   */
   public static RestException consumerInstanceNotFoundException() {
```

Comment:
I think this helper method is now just pointless complexity, so feel free to remove it and update other code unless there's some javaism here i'm not aware of.